eve/ci-values.yml

```diff
@@ -45,7 +45,7 @@ global:
 cloudserver:
   image:
     pullPolicy: Always
-  replicaCount: 0
+  replicaCount: 3
```

Comment:
Maybe should add `replicaFactor: 1` while we're at it otherwise it's 30 cloud servers in CI which is 30 more logs to sift through